ajax/libs/workbox/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "workbox",
+  "version": "3.0.0",
+  "filename": "workbox-sw.js",
+  "license": "Apache-2.0",
+  "author": "Google's Web DevRel Team",
```

Comment:
As I know we should mention real person in author field. You can open issue on lib page in order to know it